gen/pb-protodoc/flyteidl/datacatalog/index.rst

```diff
@@ -1,6 +1,10 @@
-datacatalog
-===========
+Flyte Data Catalog Service
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+These protos provide the interface definition for the Data Catalog Service. Data Catalog is a service for
```

Comment:
This file and title.rst have intersecting content regarding the catalog service. Can we cat the content from another file here ?

Reply:
i dont think so, once you have the automatic documentation stuff, we can just drop this one